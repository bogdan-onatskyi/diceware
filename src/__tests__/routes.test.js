import React from 'react';

import AppRouter from "../routes";

describe("AppRouter component", () => {
    it("should match saved snapshot", () => {
        const wrapper = shallow(
            <AppRouter/>
        );
        expect(wrapper).toMatchSnapshot();
    });

    it("should not have any props", () => {
        expect(AppRouter.propTypes).toBe(undefined);
    });
});