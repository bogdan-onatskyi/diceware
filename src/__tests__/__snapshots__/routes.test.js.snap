// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppRouter component should match saved snapshot 1`] = `
<HashRouter>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      path="/info"
    />
    <Route
      component={[Function]}
      path="/contacts"
    />
    <Route
      component={[Function]}
      path="/oops"
    />
  </Switch>
</HashRouter>
`;
